portfolio-project-matching-app/backend/dao.js

```diff
@@ -200,5 +200,47 @@ const getTechnologiesByUserId = async (userId) => {
     return technologies;
 }
 
+/*
+    TECHNOLOGY LEVEL QUERIES
+*/
```

Comment:
I mentioned on Slack that I added these. I figured since they were purely additive, they shouldn't screw anything up for anybody.